### ISSUE TITLE
Modern blur for text while app is locked via FaceID/TouchID

### ISSUE THREAD

**Issue by sigseg5** [isontheline/pro.webssh.net#1220]
# Feature description
Hi, thanks for the incredibly useful cross-platform SSH client! I have one thing I’d like to mention: when the app is locked, all the content gets blurred, and it looks questionable. It’s slightly better on macOS but absolutely not good on iOS. The text on my iPhone looks really strange. Could you please research other ways to apply a modern blur effect to the text? It definitely shouldn’t be so “blocky.”

Thx for your time!

<details> 
  <summary>Screenshots</summary>
   
![3881CD59-39E1-4DF5-9DC5-8EF27B2BA5E4_1_101_o](https://github.com/user-attachments/assets/be4ddf88-282f-4676-9210-851b88d97aad)

<img width="1092" alt="Screenshot 2024-10-14 at 23 54 55" src="https://github.com/user-attachments/assets/783f9fab-482a-4411-b532-596818993ba8">

</details>

**Comment by isontheline** [comment]
Hello @sigseg5 👋

Would you like "smoother" blur?

**Comment by sigseg5** [comment]
@isontheline yes, I think so

**Comment by isontheline** [comment]
I will take a look at it

**Comment by isontheline** [comment]
When blur is "smoother" we could use AI tools to extract content

**Comment by sigseg5** [comment]
Yes, theoretically it’s possible. Perhaps adding a checkbox with warning for ‘smoother zoom’ in the preferences would be a good solution. We could also apply a blur effect and use a ‘transition’ animation to remove the text after blur is applied. In the reverse case, we would do the opposite

**Comment by isontheline** [comment]
> Perhaps adding a checkbox with warning for ‘smoother zoom’ in the preferences would be a good solution

Good option!

**Comment by isontheline** [comment]
I'm closing this issue as the app lock is now managed by iOS and not WebSSH anymore
